readme.md

```diff
@@ -27,76 +27,140 @@ GitHub Enterprise is also supported by [authorizing your own domain in the optio
 - [**Firefox** add-on](https://addons.mozilla.org/en-US/firefox/addon/refined-github-/)
 - Opera - Use [this Opera extension](https://addons.opera.com/en/extensions/details/download-chrome-extension-9/) to install the Chrome version.
 
-
 ## Highlights
 
-- [Mark issues and pull requests as unread](https://cloud.githubusercontent.com/assets/170270/18231475/bdf83e26-72e4-11e6-958f-9ce9431d80eb.png) *(They will reappear in Notifications)*
+<table>
+    <tr>
+        <th>
+            Dashboard cleanup
+        </th>
+    </tr>
+    <tr><!-- no zebra stripes thx --></tr>
+    <tr>
+        <td>
+            <img src="media/screenshot-dashboard.png">
+        </td>
+    </tr>
+</table>
+
+<table>
```

Comment:
Glad you asked. Unfortunately not for a few reasons:

- no `width: 50%` support
- no single-column table support (which is nice because the borders wrap the title and images nicely IMHO)
- they look worse than HTML once the content gets a bit longer because you can't wrap it, so you'll end up with stuff like:
    
   ```md
    Mark issues and pull requests as unread<br> *(They will reappear in Notifications)* | Preserves the original Markdown when you copy text from comments
    --- | ---
    <img width="245" src="https://user-images.githubusercontent.com/1402241/27847663-963b7d7c-6171-11e7-9470-6e86d8463771.png"> | <img width="400" src="https://user-images.githubusercontent.com/170270/27501181-0485c5d0-586c-11e7-91ad-2d0a3537b0e2.gif">
    ```